### DISCUSSION TITLE
Is is possible to change control's size by easily drag like VS,...

### DISCUSSION THREAD

**Discussion by ruihuasun** [microsoft/microsoft-ui-xaml#8158] [Q&A] [answered]
GridSplitter in Windows Community Toolkit doesn't seem to work with winui3, and it doesn't look pretty.

**Comment by castorix** [comment, marked as answer]
A method can be to use a Thumb control, like in the WPF GridSplitter :
(_VerticalAlignment_="Stretch" inside a Grid in this test)

![GridSplitter](https://user-images.githubusercontent.com/22345506/218095492-0663d0d7-c6ea-4380-983c-cfd73c3d2d79.gif)

**Reply by ruihuasun** [reply]
Thanks! I'll go into it.

**Reply by MikeHillberg** [reply]
The community Toolkit is working for me with WinUI3 (I'm using CommunityToolkit.WinUI.UI.Controls 7.1.2). The two issues I haven't debugged yet are:
* The cursor doesn't change when over, only when dragging (even with `ChangeOnSplitterHover` set)
* I don't want the background color. You can set the Background to Transparent, but that gets overriden on mouse-over

**Reply by ruihuasun** [reply]
> The community Toolkit is working for me with WinUI3 (I'm using CommunityToolkit.WinUI.UI.Controls 7.1.2). The two issues I haven't debugged yet are:
> 
> * The cursor doesn't change when over, only when dragging (even with `ChangeOnSplitterHover` set)
> * I don't want the background color. You can set the Background to Transparent, but that gets overriden on mouse-over

@MikeHillberg Yes, the first point happened to be too. 
If the grid layout is simple like 1 * 3, 3 * 1, the community Toolkit GridSplitter works. However, when it gets complicated, like 1 col in first row, 3 cols in second row, 2 cols in third row, the control seems incorrect.
